lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -44,5 +44,5 @@ echo -e "\033[32m ✓\033[39m Report renderer files copied."
 cp -pPR "$lh_bg_js" "$lh_worker_dir/lighthouse-dt-bundle.js"
 echo -e "\033[96m ✓\033[39m (Potentially stale) lighthouse-dt-bundle copied."
 
-# copy report generator
-cp dist/report-generator.js $fe_lh_dir
+# copy report generator + cached resources
+cp -r dist/dt-resources/ $fe_lh_dir
```

Comment:
kinda unrelated but the power of the trailing `/` always terrifies me in `cp` commands...

this means we want `dt-resources` to now be at `$fe_lh_dir/dt-resources` or we want `$fe_lh_dir` to *be* `dt-resources`?

Reply:
added comment to clarify. `-R` + trailing `/` on the source copies the contents of the folder. If the destination doesn't exist or is not a folder, it fails.

Reply:
ah OK, so it's equivalent to `dist/dt-resources/*` in our case, right?

Reply:
yes, but they'd act differently in some cases like (had to look this up):

* glob does not capture hidden files
* there is only one file matching the glob + destination folder does not exist => single file is copied as a file. no destination directory created
* if there are multiple files matching the glob + destination folder does not exist => errors

Reply:
of course, the above is a consequence of the shell expanding the glob, and the difference between `cp -r f1 f2 sd` and `cp -r f1 sd`